spec/features/user_can_submit_posts_spec.rb

```diff
@@ -10,4 +10,15 @@
     click_button 'Submit'
     expect(page).to have_content('Hello, world!')
   end
+
+  scenario 'Posts are shown in order of creation' do
+    visit '/posts'
+    click_link 'New post'
+    fill_in 'Message', with: 'I am second'
+    click_button 'Submit'
+    click_link 'New post'
+    fill_in 'Message', with: 'I am first'
+    click_button 'Submit'
+    expect(page.all('p')[0]).to have_content 'I am first'
```

Comment:
Potentially could break in future if we had other `<p>` tags to the page. Potentially can check this in the `message` class